code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -814,6 +814,17 @@
 	construction_time = 10 SECONDS
 	category = list("Exosuit Equipment")
 
+/datum/design/mech_armor_plate
+	name = "Exosuit Mining Armor Plate"
+	desc = "Exosuit-mounted armor booster."
```

Comment:
Description is the same as the "ranged armor booster module", change it to something different and more descriptive.

Reply:
done